### ISSUE TITLE
Big Sur compatibility

### ISSUE THREAD

**Issue by revanmj** [Mortennn/Dozer#134]
Just wanted to let you know, that while Dozer 4.0 works fine in Big Sur Public Beta, version 4.2 beta doesn't.

It launches without any crash, but dot icons are not visible (nor clickable) on menubar.

**Comment by umputun** [comment]
both Dozer 4.0 and 4.2 don't work for me on macOS 11 beta. Dots are not visible

**Comment by revanmj** [comment]
In 4.0 dots were not visible for me once, but reboot fixed this. 

Unfortunately, I've noticed that icon placement is not remembered (apps that were closed and relaunched always are hidden as they are always put at the end, no matter how many times you moved them past the dot to keep them visible).

**Comment by Stanzilla** [comment]
The icon placement thing is an apple bugs, please report it via the feedback tool, I did so as well.

**Comment by revanmj** [comment]
Also submitted this to Apple.

**Comment by lucky13820** [comment]
4.2.0(30) works for me on Big Sur Public Beta. The icon placement thing is really annoying though.

**Comment by vuza** [comment]
Worked for me on Big Sur Beta 6 until first reboot. Tried 4.0.0 and 4.2.0, tried deleting/re-installing :/

How can I help to track down this issue?

**Comment by revanmj** [comment]
9th Big Sur beta and OS still does not remember icon position between app launches.

I'm afraid they are not going to fix this any time soon :(

**Comment by lucky13820** [comment]
Yeah, I think so too. This is sad, hopefully, someone can find a workaround.

**Comment by Stanzilla** [comment]
Bartender4 is out now and it has an option to remember and then restore your icon positions, works pretty well

**Comment by lucky13820** [comment]
Where did you get Bartender 4? I don't see any update.

**Comment by Stanzilla** [comment]
From https://www.macbartender.com/Bartender4/ it's not linked on the main page

**Comment by lucky13820** [comment]
Thank you!

**Comment by foo2k** [comment]
Is Dozer still not working on Big Sur (final release)?

**Comment by frbuccoliero** [comment]
For me (final release 11.0.1) dozer works fine but forgets the position of the icons after reboot. If i move them they came back as macos wants them, a few hidden, a few shown.

**Comment by tirannosario** [comment]
> For me (final release 11.0.1) dozer works fine but forgets the position of the icons after reboot. If i move them they came back as macos wants them, a few hidden, a few shown.

Same :(

**Comment by Stanzilla** [comment]
Like I said, Bartender4 has a nice option that fixes that.

**Comment by vidalpascual** [comment]
> For me (final release 11.0.1) dozer works fine but forgets the position of the icons after reboot. If i move them they came back as macos wants them, a few hidden, a few shown.

Same here... :(

**Comment by marnovo** [comment]
More detailed info about the Big Sur issue: https://github.com/feedback-assistant/reports/issues/151

**Comment by cnLGMing** [comment]
> 

Same here... :(

**Comment by psy0ch** [comment]
Just updated to 11.0.1 and Dozerr wont hide any Icons at all. 
I´ve installed the Release 4.2.0 after trying Version 4.0.0 of Dozerr, sadly it didnt work. 
The Dots retain their positions i´ve assigned to them, but they dont hide the icons in between themselves.

**Comment by matdave** [comment]
> 
> 
> Just updated to 11.0.1 and Dozerr wont hide any Icons at all.
> I´ve installed the Release 4.2.0 after trying Version 4.0.0 of Dozerr, sadly it didnt work.
> The Dots retain their positions i´ve assigned to them, but they dont hide the icons in between themselves.

Same issues here.

**Comment by revanmj** [comment]
I just got Apple response on the "not remembering icons moved with Cmd + drag" issue:

> There are changes in the latest update, build 20C5048k, that may have resolved this issue.

I'm on stable now and don't want to install beta, so if anyone could confirm this :)

**Comment by marnovo** [comment]
Confirmed, fixed on macOS 11.1 beta 1 (20C5048k) and continues fixed on beta 2 (20C5061b).

**Comment by uzair-mts** [comment]
> > For me (final release 11.0.1) dozer works fine but forgets the position of the icons after reboot. If i move them they came back as macos wants them, a few hidden, a few shown.
> 
> Same :(

Same issue.

**Comment by Bustycat** [comment]
It is confirmed as a macOS bug and fixed on 11.1 (20C69).

**Comment by cherryblossom000** [comment]
I just updated to macOS Big Sur 11.2 and the Dozer icons are not showing when the menu bar items are hidden. Opening the Dozer application shows the extra menu bar items and the dots, and everything else works, but once I click on the first dot the menu bar items are hidden and the Dozer dot isn't visible.

**Comment by fuzzy76** [comment]
First of all, that is not what this issue is about, so it should probably be a separate issue.

Second, are you sure it's not just this setting?

![Skjermbilde 2021-02-05 kl  08 30 19](https://user-images.githubusercontent.com/218995/107003145-9b172600-678c-11eb-90ea-478e72bd6544.png)

**Comment by cherryblossom000** [comment]
@fuzzy76 Yes, that setting is disabled. I have also tried turning that on and off.

Sorry for cluttering this issue — I'll create a separate one for my problem.

**Comment by gingerbeardman** [comment]
Dozer 4.2.0 works for me in macOS 11.3.1

**Comment by gerrytan** [comment]
I just installed this for the first time in big sur, it looks broken.

https://user-images.githubusercontent.com/2316198/155448691-114dbb64-5aad-44d0-a63d-91efcecd511e.mov

**Comment by gingerbeardman** [comment]
@gerrytan your video doesn't show it broken, but rather that you don't yet understand how to use the app. You need to drag the dots to split the icons into groups. See the ["Usage" section of the README](https://github.com/Mortennn/Dozer/blob/master/README.md#-usage)

For the record, the app works just fine on Big Sur and Monterey

**Comment by gerrytan** [comment]
Ah I get it now. Yes it works fine. Thx @gingerbeardman .
